boa3_test/tests/compiler_tests/test_interop/test_runtime.py

```diff
@@ -426,20 +426,30 @@ def test_invocation_counter_cant_assign(self):
 
     def test_get_notifications(self):
         path = self.get_contract_path('GetNotifications.py')
-
         engine = TestEngine()
+
+        from boa3.constants import MANAGEMENT_SCRIPT
+
         result = self.run_smart_contract(engine, path, 'without_param', [])
-        self.assertEqual([], result)
+        self.assertEqual(1, len(result))
+        self.assertEqual(MANAGEMENT_SCRIPT, result[0][0])
+        self.assertEqual('Deploy', result[0][1])
```

Comment:
import `boa3.constants` in the beginning of the file and use `constants.MANAGEMENT_SCRIPT` here
also use variables no make the code clearer
```suggestion
        self.assertEqual(1, len(result))
        self.assertEqual(2, len(result[0]))
        event_script, event_name = result[0][:1]
        self.assertEqual(constants.MANAGEMENT_SCRIPT, event_script)
        self.assertEqual('Deploy', event_name)
```